scripts/tracker.py

```diff
@@ -102,17 +111,14 @@ def is_ok_go_next(self):
         dx2 = (self.cur_pos.pose.pose.position.x - self.sp.position.x) ** 2
         dy2 = (self.cur_pos.pose.pose.position.y - self.sp.position.y) ** 2
         dz2 = (self.cur_pos.pose.pose.position.z - self.sp.position.z) ** 2
-        return dx2 + dy2 + dz2 <= 1
+        return dx2 + dy2 + dz2 <= self.track_distance
 
     def update_sp(self):
         if len(self.sp_list) == 0:
             return
         if self.sp != None and not self.is_ok_go_next():
             return
-        while len(self.sp_list) > 0 and (
-            dist(self.cur_pos.pose.pose.position, self.sp_list[0].position) < 2.0
-            or self.sp_list[0].position.z < 1.0
-        ):
+        while len(self.sp_list) > 0 and self.sp_list[0].position.z < 0.3:
```

Comment:
0.3 要不要给个名字比如 min_height?
```suggestion
        while len(self.sp_list) > 0 and self.sp_list[0].position.z < self.min_height:
```

Reply:
给了个可读性更好的名字 dangerous_height